vfx/gl/texture_program.cc

```diff
@@ -0,0 +1,60 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "vfx/gl/texture_program.h"
+
+#include <memory>
+
+#include "ui/gl/gl_bindings.h"
+
+namespace vfx {
+namespace {
+
+const char kVertexShaderSource[] = R"GLSL(
+uniform mat4 u_transform;
+attribute vec4 a_position;
+attribute vec4 a_color;
+attribute vec2 a_tex_coord;
+
+varying vec2 v_tex_coord;
+varying vec4 v_color;
+
+void main() {
+  gl_Position = u_transform * a_position, 1.0;
```

Comment:
So are you aware that you just set gl_Position to vec4(1.0) (i.e. <1,1,1,1>) regardless of whats in u_transform and a_position? Not sure if your doing this on purpose so it doesn't optimize u_transform and a_position out of existence (and therefore make it impossible to bind values to them), but I feel like this guy should at least get a comment in case someone is reading this code trying to figure out why they cant see any of their vertices. See [https://en.wikipedia.org/wiki/Comma_operator](https://en.wikipedia.org/wiki/Comma_operator)